addons/mail/static/src/discuss/call/common/call_invitations.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="discuss.CallInvitations">
-        <div class="o-discuss-CallInvitations position-absolute top-0 end-0 d-flex flex-column p-2">
+        <div t-if="store.discuss.ringingThreads.length > 0" class="o-discuss-CallInvitations position-absolute top-0 end-0 d-flex flex-column p-2">
```

Comment:
This fix will only fix the bug if there is no call in progress, it should be done differently, for example adding me-3 instead.
Or report the bug to chrome??!

Reply:
or right: something in css instead of `end-0`

Reply:
This main component should be inert when there's no ringing threads.
It could be find with `t-foreach` but due to `p-2` (about 16px square), absolute positioning could prevent main page UI. Main components are after `o_action_manager` so this is very likely to prevent main UI clicks.

`me-3` is not a good solution because this could prevent click on systray items.